com.mojang.Minecraft.json

```diff
@@ -109,6 +111,62 @@
                     "path": "com.mojang.Minecraft.appdata.xml"
                 }
             ]
+        },
+        
+        {
+            "name": "extra-cmake-modules",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+              "-DBUILD_TESTING=OFF"
+            ],
+            "sources": [
+              {
+                "type": "archive",
+                "url": "https://download.kde.org/stable/frameworks/5.74/extra-cmake-modules-5.74.0.tar.xz",
+                "sha256": "71406067bcd99ac106e0e3bfbb073653b18fd6d01039c0298d9767680977364f"
+              }
+            ]
+          },
+          {
+            "name": "libdecor",
+            "buildsystem": "meson",
+            "builddir": true,
+            "sources": [
+              {
+                "type": "archive",
+                "url": "https://gitlab.gnome.org/jadahl/libdecoration/-/archive/master/libdecoration-master.tar.gz",
```

Comment:
Using archive made from latest master is not a good idea. ;-)

Reply:
Also, it looks like the repository has moved to https://gitlab.gnome.org/jadahl/libdecor - that has a tagged release you could use. ;-)

Reply:
This took me a minute to figure out, turns out all I needed to do was update the libdecor patch (yay juggling patches).

I do hope to try and debug the issues on the to do list at some point, because that's probably a much better use of time, especially the last one. Since libdecor got into SDL hopefully the patches get reviewed sooner than later.